lib/cors-anywhere.js

```diff
@@ -246,7 +270,20 @@ function getHandler(options, proxy) {
   };
 
   return function(req, res) {
+    // Set the options to apply when generating CORS headers on the request
+    req.corsOptions = {
+      allowedMethodsString: corsAnywhere.allowedMethodsString = corsAnywhere.allowedMethods.join(','),
```

Comment:
Change this to `allowedMethodsString: corsAnywhere.allowedMethodsStrings`

and put `corsAnywhere.allowedMethodString = ....join();` right after the if-block that copies external options to `corsAnywhere`. (This is a small optimization because the value is not going to change.)